docs/DesignMDIngestion.md

```diff
@@ -0,0 +1,63 @@
+# *Generic Metadata Ingestion*
+
+## Description
+
+This is the design document discussing the ingestion process of existing metadata
+into Zenko with MongoDB.
+
+The current (primary) use case is to ingest existing metadata from the RING and
+S3 Connector; future use cases will include ingesting metadata from existing storage
+solutions
+on AWS or Azure.
+
+This specific development will allow Zenko instances to copy and ingest existing
+metadata,
+copying the information from the raft logs completely and allowing MongoDB to be
+a parallel metadata database.
+
+## Requirements
+
+* Copy all existing metadata from pre-existing storage solutions to MongoDB in Zenko.
+
+* Allow MongoDB to be used in parallel with the existing metadata servers with S3
+  Connector.
+
+## Design
+
+*Note: current design focuses primarily on RING and Zenko*
```

Comment:
I think maybe adding a line break would be helpful when this is viewed as markdown. Otherwise it reads as a single sentence.

Reply:
Thank you for the review!